src/Framework/src/Microsoft.AspNetCore.App.Runtime.csproj

```diff
@@ -96,7 +96,8 @@ This package is an internal implementation of the .NET Core SDK and is not meant
     <CrossgenToolPackagePath Condition=" '$(TargetRuntimeIdentifier)' == 'linux-arm64' OR '$(TargetRuntimeIdentifier)' == 'linux-musl-arm64' ">x64_arm64\$(CrossgenToolPackagePath)</CrossgenToolPackagePath>
     <CrossgenToolPackagePath Condition=" '$(TargetRuntimeIdentifier)' == 'win-arm' ">x86_arm\$(CrossgenToolPackagePath)</CrossgenToolPackagePath>
 
-    <RuntimePackageRoot>$([System.IO.Path]::Combine('$(NuGetPackageRoot)', 'runtime.$(RuntimeIdentifier).microsoft.netcore.app', '$(MicrosoftNETCoreAppPackageVersion)'))</RuntimePackageRoot>
+    <!--TODO this seems fishy. -->
+    <RuntimePackageRoot>$([System.IO.Path]::Combine('$(NuGetPackageRoot)', 'runtime.$(RuntimeIdentifier).microsoft.netcore.app', '$(MicrosoftNETCoreAppRuntimewinx64PackageVersion)'))</RuntimePackageRoot>
```

Comment:
Is the package name still 'runtime.$(RuntimeIdentifier).microsoft.netcore.app'?

Reply:
From what I can tell, both are produced at the moment, but both exist in my .nuget folder. I believe the right name is:
`microsoft.netcore.app.runtime.$(RuntimeIdentifier)`

Reply:
yes, `microsoft.netcore.app.runtime.$(RuntimeIdentifier)` is the right package name to use now.